app/domain/healthchecks/aggregations.rb

```diff
@@ -0,0 +1,23 @@
+module Healthchecks
+  class Aggregations
+    include Concerns::Deactivable
+
+    def name
+      :aggregations
+    end
+
+    def status
+      aggregations.values.all? { |agg| !agg.nil? } ? :ok : :critical
+    end
+
+    def message
+      "ETL :: no aggregations of metrics for yesterday" if status == :critical
+    end
+
+  private
+
+    def aggregations
+      @aggregations ||= Finders::Aggregations.new.between(from: Date.yesterday, to: Date.today).run
```

Comment:
Not sure about the Aggregations. We would need to ensure that we have monthly aggregations. I would count on this table filtering by creation time.

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/1153/commits/69c1cacfa050fc37d3341465f047b3e0f3edec4c